flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/common/MatchRecognizeTestPrograms.java

```diff
@@ -181,6 +203,47 @@ public class MatchRecognizeTestPrograms {
                     .runSql(getEventTimeSql("ORDER BY rowtime, sequence_num"))
                     .build();
 
+    static final SourceTestStep SOURCE_WITH_OUT_OF_ORDER_DATA =
```

Comment:
Why is this in the default scope? Can't it be `private` if `public` is not necessary?